ostd/src/mm/kspace/mod.rs

```diff
@@ -17,17 +17,17 @@
 //! +-+ <- 0xffff_ffff_8000_0000
 //! | |
 //! | |         Unused hole.
-//! +-+ <- 0xffff_ff00_0000_0000
-//! | |         For frame metadata, 1 TiB.
-//! | |         Mapped frames are untracked.
-//! +-+ <- 0xffff_fe00_0000_0000
-//! | |         For vm alloc/io mappings, 1 TiB.
-//! | |         Mapped frames are tracked with handles.
-//! +-+ <- 0xffff_fd00_0000_0000
+//! +-+ <- 0xffff_e100_0000_0000
+//! | |         For frame metadata, 1 TiB. Mapped frames are untracked.
+//! +-+ <- 0xffff_e000_0000_0000
+//! | |         For [`kva::Kva`], 16 TiB. Mapped pages are tracked with handles.
+//! +-+ <- 0xffff_d000_0000_0000
+//! | |         For [`kva::Kva`], 16 TiB. Mapped pages are untracked.
+//! +-+ <- the middle of the higher half (0xffff_c000_0000_0000)
```

Comment:
We made the kva area very small (1T) on top because that TDX would have very high IoMem mappings using linear mappings. Only After you uses `Kva` to refactor `IoMem` you can make the address space layout like this. I encourage you to implement it in this PR.